packages/salesforcedx-sobjects-faux-generator/src/transformer/sobjectTransformer.ts

```diff
@@ -133,7 +132,7 @@ export class SObjectTransformer {
         this.result.error = { message, stack };
       },
 
-      sfdxPath: path.join(projectPath, SFDX_DIR),
+      sfdxPath: projectPaths.stateFolder(),
```

Comment:
This seems unnecessary, but it's touched throughout the module so I didn't want to refactor it now.